src/app/libs/notification/services/notification.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { NotificationComponent } from '../components/notification.component';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class NotificationService {
+
+    constructor (private _snackBar: MatSnackBar) {
+    }
+
+    showError (message: string, duration: number = 5000) {
```

Comment:
you don't need explicit type for `duration` here because it is detected from value you are assigning